src/main/java/thePackmaster/cardmodifiers/lockonpack/GlockOnModifier.java

```diff
@@ -0,0 +1,38 @@
+package thePackmaster.cardmodifiers.lockonpack;
+
+import basemod.abstracts.AbstractCardModifier;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.orbs.AbstractOrb;
+import com.megacrit.cardcrawl.powers.FocusPower;
+import com.megacrit.cardcrawl.powers.LockOnPower;
+import thePackmaster.util.Wiz;
+
+public class GlockOnModifier extends AbstractCardModifier {
+
+    @Override
+    public float modifyDamage(float damage, DamageInfo.DamageType type, AbstractCard card, AbstractMonster target) {
+        return damage + Wiz.getLogicalPowerAmount(AbstractDungeon.player, FocusPower.POWER_ID);
+    }
+    @Override
+    public float modifyDamageFinal(float damage, DamageInfo.DamageType type, AbstractCard card, AbstractMonster target) {
+        if (target == null) return super.modifyDamage(damage, type, card, target);
```

Comment:
Need an explanation for what's happening here. Wouldn't this just not work for AoE attacks? 
What's the leftover var below for? Some weird form of casting?

Reply:
I could either do maths in card itself with ap/ccd, or make an inherent cardmod. This is the latter. Only intended to be used on one card, but, well, if you wish.
The leftover var is because normal damage is calculated with floats, but lock-on only runs ints. Will update a bit too.